unittest/unicharset_test.cc

```diff
@@ -62,7 +77,7 @@ TEST(UnicharsetTest, Multibyte) {
   EXPECT_EQ(u.size(), 9);
   EXPECT_EQ(u.unichar_to_id("\u0627"), 3);
   EXPECT_EQ(u.unichar_to_id("\u062c"), 4);
-  // The first two bytes of this string is \u0627, which matches id 3;
+  // The first two bytes of this std::string is \u0627, which matches id 3;
```

Comment:
That change was not needed.

Reply:
You are right. I had done a replace for ' string ' by ' std::string '.